pkg/jx/cmd/delete_application.go

```diff
@@ -312,29 +312,28 @@ func (o *DeleteApplicationOptions) waitForGitOpsPullRequest(env *v1.Environment,
 			if pr.Merged != nil && *pr.Merged {
 				log.Infof("Request %s is merged!\n", util.ColorInfo(pr.URL))
 				return nil
+			}
+			if pr.IsClosed() {
```

Comment:
is this applications or apps?   I am a little confused as to what is this PR is trying to acheive - and exepected it from the first few files to be "deleting (aka uninstalling) apps"

Reply:
Hmm, possibly something went wrong, I meant to do the unalias as a single commit. Let me look.

Reply:
Oh, this looks like a reformat of a file that got caught up. Let me remove it.